exporter/clickhousemetricsexporter/clickhouse.go

```diff
@@ -198,7 +199,7 @@ func (ch *clickHouse) GetDBConn() interface{} {
 	return ch.conn
 }
 
-func (ch *clickHouse) Write(ctx context.Context, data *prompb.WriteRequest, metricNameToMeta map[string]base.MetricMeta) error {
+func (ch *clickHouse) Write(ctx context.Context, exporterID uuid.UUID, data *prompb.WriteRequest, metricNameToMeta map[string]base.MetricMeta) error {
```

Comment:
You could make id part of the `NewClickHouse` instead of update the write interface?